app/src/main/java/com/globant/example/mentorapp/data/remote/ApiClient.java

```diff
@@ -0,0 +1,21 @@
+package com.globant.example.mentorapp.data.remote;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Created by oscar.amezquita on 7/06/2017.
+ */
+
+public class ApiClient {
+
+    private static Retrofit retrofit = null;
+
+    public static Retrofit getClient() {
```

Comment:
Use a Dagger Module to provide the API object and inject that dependency in your presenter